src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -51,6 +51,9 @@
  * {@link TokenTypes#INDEX_OP INDEX_OP}
  * specially from other tokens. Actually it is checked that there is
  * no whitespace before this tokens, not after them.
+ * {@link TokenTypes#ANNOTATIONS ANNOTATIONS}
```

Comment:
Check does not ignore annotations, it is ignoring spaces.

Reply:
fixed